src/appleseed.shaders/src/appleseed/as_blackbody.osl

```diff
@@ -271,14 +271,14 @@ shader as_blackbody
 
         transmittance = 1.0 - clamp(Kr, 0.0, 1.0);
 
-        out_color += in_specular_amount *
+        out_color += in_specular_weight *
             as_glossy(
                 "ggx",
                 Nn,
                 vector(0), // tangent vector
                 in_specular_roughness,
                 0.0, // specular spread
-                0.0, // anisotropy amount
+                0.0, // anisotropy weight
```

Comment:
That's an example of _amount_ being a more proper term than _weight_, in my opinion.